apps/src/lib/levelbuilder/rubrics/LearningGoalItem.jsx

```diff
@@ -30,14 +30,22 @@ export default function LearningGoalItem({deleteItem}) {
                 />
               </label>
             </div>
-            <label>
+            <label style={styles.labelAndInput}>
               Use AI to assess
               <input
                 type="checkbox"
                 checked={aiEnabled}
                 onChange={handleCheckboxChange}
                 style={styles.checkbox}
               />
+              <span
+                style={
+                  aiEnabled ? styles.checkboxChecked : styles.checkboxBlank
+                }
+                aria-hidden="true"
+              >
+                &#10003;
```

Comment:
Mind adding a comment explaining what this means? (I googled it so I know its a checkmark)

Reply:
Or for extra bonus readability, why not just put the unicode checkmark directly in the code file?

Reply:
Yes, I added some comments.

Reply:
Updated with the unicode check mark